### PR TITLE
rebase bot - add ghstack config

### LINKED ISSUE #78539: Rebasing with bot doesn't work with ghstack
Looks like we are not configuring ghstack correctly?

https://github.com/pytorch/pytorch/pull/78287#issuecomment-1142371158

### PR DESCRIPTION
**Pull request by clee2000**
Fixes #78539
testing pr: https://github.com/clee2000/random-testing/pull/14

### CODE REVIEW

File: .github/scripts/tryrebase.py [hunk 1]
```diff
@@ -48,6 +48,14 @@ def rebase_ghstack_onto(pr: GitHubPR, repo: GitRepo, dry_run: bool = False, stab
 
     repo.fetch(orig_ref, orig_ref)
     repo._run_git("rebase", onto_branch, orig_ref)
+
+    os.environ["OAUTH_TOKEN"] = os.environ["GITHUB_TOKEN"]
+    with open('.ghstackrc', 'w+') as f:
+        f.write('[ghstack]\n' +
+                "github_url=github.com\n" +
+                "github_username=clee2000\n" +
```
Comment (janeyx99) [inline .github/scripts/tryrebase.py:56]: Should this always be you?

Comment (clee2000) [reply .github/scripts/tryrebase.py]: one days, im going to remember to change all of these

**Review by janeyx99 — APPROVED** [review]

**Comment by janeyx99** [comment]
Do you have a test plan btw showing that this works?

**Comment by clee2000** [comment]
> Do you have a test plan btw showing that this works?

added a link to my testing pr

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**